utils/src/field_helpers.rs

```diff
@@ -152,7 +152,7 @@ mod tests {
         let field_hex = "f2eee8d8f6e5fb182c610cae6c5393fce69dc4d900e7b4923b074e54ad00fb36";
         assert_eq!(
             BaseField::to_hex(
-                BaseField::from_hex(field_hex).expect("Failed to deserialize field hex")
+                &BaseField::from_hex(field_hex).expect("Failed to deserialize field hex")
```

Comment:
Now users don't need to give up the ownership of BaseField so that it can be reused for other purposes without cloning which are common in business logic